coalib/bears/PythonRequirement.py

```diff
@@ -0,0 +1,33 @@
+from coalib.bears.PackageRequirement import PackageRequirement
+
+
+class PythonRequirement(PackageRequirement):
+
+    def __init__(self, package, version=""):
+        """
+        :param package: A string with the name of the package to be installed.
+        :param version: A number that contains the version. Leave empty to
+                        install latest.
+        """
+        PackageRequirement.__init__(self, 'pip', package, version)
+
+    @classmethod
+    def multiple(cls, *args):
+        """
```

Comment:
Please also a brief description what this method is for with a doctest that demonstrates the usage.
Also add `:return:` description^^